<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RenderingWidget</class>
 <widget class="QWidget" name="RenderingWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>156</width>
    <height>239</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="labelRendering">
     <property name="text">
      <string>&lt;b&gt;Deferred Shading&lt;/b&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="boxDiffuseR">
     <property name="text">
      <string>Diffuse</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="boxNormalsR">
     <property name="text">
      <string>Normals</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="boxSpecularR">
     <property name="text">
      <string>Specular</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="lineRendering1">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="boxNormalMapping">
     <property name="text">
      <string>Normal Mapping</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="boxHDR">
     <property name="text">
      <string>HDR</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="boxBloom">
     <property name="text">
      <string>Bloom</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="lineRendering2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="buttonRendering">
     <property name="text">
      <string>Reload shaders</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
